certificate-authority/store/cqldb/signingRecords_test.go

```diff
@@ -0,0 +1,579 @@
+		})
+	}
+}
+
+func TestStoreDeleteExpiredRecords(t *testing.T) {
+	const id = "9d017fad-2961-4fcc-94a9-1e1291a88ffc"
+	date := time.Now().Add(time.Second * 2)
+
+	s, cleanUpStore := test.NewCQLStore(t)
+	defer cleanUpStore()
+
+	ctx := context.Background()
+	err := s.CreateSigningRecord(ctx, &store.SigningRecord{
+		Id:           id,
+		Owner:        "owner",
+		CommonName:   "commonName",
+		PublicKey:    "publicKey",
+		CreationDate: date.UnixNano(),
+		Credential: &pb.CredentialStatus{
+			CertificatePem: "certificate",
+			Date:           date.UnixNano(),
+			ValidUntilDate: date.UnixNano(),
+		},
+	})
+	require.NoError(t, err)
+	var h testSigningRecordHandler
+	err = s.LoadSigningRecords(ctx, "owner", nil, h.Handle)
+	require.NoError(t, err)
+	require.Len(t, h.lcs, 1)
+	time.Sleep(time.Second * 3)
+	_, err = s.DeleteNonDeviceExpiredRecords(ctx, time.Now())
+	require.Error(t, err)
+	require.Equal(t, store.ErrNotSupported, err)
+	var h1 testSigningRecordHandler
+	err = s.LoadSigningRecords(ctx, "owner", nil, h1.Handle)
+	require.NoError(t, err)
+	require.Len(t, h1.lcs, 0)
+}
+
+type testSigningRecordHandler struct {
+	lcs pb.SigningRecords
+}
+
+func (h *testSigningRecordHandler) Handle(ctx context.Context, iter store.SigningRecordIter) (err error) {
+	for {
+		var sub store.SigningRecord
+		if !iter.Next(ctx, &sub) {
+			break
+		}
+		h.lcs = append(h.lcs, &sub)
+	}
+	return iter.Err()
+}
+
+func TestStoreLoadSigningRecords(t *testing.T) {
+	const id = "9d017fad-2961-4fcc-94a9-1e1291a88ffc"
+	const id1 = "9d017fad-2961-4fcc-94a9-1e1291a88ffd"
+	const id2 = "9d017fad-2961-4fcc-94a9-1e1291a88ffe"
+	date := time.Now().Add(time.Hour)
+	upds := pb.SigningRecords{
+		{
+			Id:           id,
+			Owner:        "owner",
+			CommonName:   "commonName",
+			PublicKey:    "publicKey",
+			DeviceId:     hubTest.GenerateDeviceIDbyIdx(1),
+			CreationDate: date.UnixNano(),
+			Credential: &pb.CredentialStatus{
+				CertificatePem: "certificate",
+				Date:           date.UnixNano(),
+				ValidUntilDate: date.UnixNano(),
+			},
+		},
+		{
+			Id:           id1,
+			Owner:        "owner",
+			CommonName:   "commonName1",
+			CreationDate: date.UnixNano(),
+			PublicKey:    "publicKey",
+			DeviceId:     hubTest.GenerateDeviceIDbyIdx(2),
+			Credential: &pb.CredentialStatus{
+				CertificatePem: "certificate",
+				Date:           date.UnixNano(),
+				ValidUntilDate: date.UnixNano(),
+			},
+		},
+		{
+			Id:           id2,
+			Owner:        "owner",
+			CommonName:   "commonName2",
+			CreationDate: date.UnixNano(),
+			PublicKey:    "publicKey",
+			DeviceId:     hubTest.GenerateDeviceIDbyIdx(3),
+			Credential: &pb.CredentialStatus{
+				CertificatePem: "certificate",
+				Date:           date.UnixNano(),
+				ValidUntilDate: date.UnixNano(),
+			},
+		},
+	}
+
+	lcs := upds
+
+	type args struct {
+		owner string
+		query *store.SigningRecordsQuery
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
```

Comment:
unused, seems that there is no way of triggering error with just a query. You could create a handler that returns an error, but `signingRecords.go` has already 90+% code coverage, so I think we can just remove this member

Reply:
Removed